### ISSUE TITLE
Etnofarmakolog vs Etnofarmasi

### ISSUE THREAD

**Issue by Hypsibius** [realfagstermer/realfagstermer#127]
Treng ikkje begge. Foreslår Etnofarmasi, BR etnofarmakolog

**Comment by Hypsibius** [comment]
etnofarmakologi, skulle det vere...

**Comment by Hypsibius** [comment]
Men kikkar vidare nedover lista: korleis skal desse forhalde seg til etnomedisin (se Folkemedisin)?

**Comment by danmichaelo** [comment]
Hvis definisjonene på Wikipedia (hjelpehjernen) stemmer sånn noenlunde synes jeg det ser ut som disse ikke skal forholde seg til hverandre i det hele tatt.

Etnomedisin eller folkemedisin er sammenliknende studium av hvordan folk klassifiserer, oppfatter og søker måter å helbrede sykdommer.

Etnofarmakologi er studiet av medisinske planters virkninger, og hvordan bruken/virkningene av dem varierer mellom ulike kulturer og etniske grupper.

**Comment by Hypsibius** [comment]
Hm, synest fortsatt etnofarmakologi høyrest ut som om det overlappar breitt med siste tredjedelen av etnomedisin, gitt at forsøk på å helbrede skrekkeleg ofte involverer plantar ;-) Men åkke som treng vi gjerne begge termar

**Comment by danmichaelo** [comment]
Eh jo, jeg ser hva du mener nå :) Lurer forresten på om WP-definisjonen er litt upresis, og at det ville vært mer riktig å si at "Etnomedisin er studiet av folkemedisin" jfr. engelsk Wikipedia... 

Men uansett, når jeg ser på litteraturbelegget vårt (UBO) for [Folkemedisin](http://katapi.biblionaut.net/documents?q=real:Folkemedisin&show=sh), [Etnofarmasi](http://katapi.biblionaut.net/documents?q=real:Etnofarmasi&show=sh), [Etnofarmakologi](http://katapi.biblionaut.net/documents?q=real:Etnofarmakologi&show=sh) får jeg litt inntrykk av at det egentlig ikke er et klart nok skille mellom disse til at vi klarer å opprettholde det i indekseringsarbeidet..

Er `Folkemedisin BF Etnofarmasi, BF Etnofarmakologi BF Etnomedisin` å ta for hardt i? Kan evt. spørre Bente som har ansvaret for farmasi her

**Comment by Hypsibius** [comment]
Ja, spør henne. Ingunn er ikkje heilt tilgjengeleg for tida....

**Comment by Hypsibius** [comment]
https://github.com/realfagstermer/realfagstermer/issues/743

**Comment by danmichaelo** [comment]
Reminder to self: spørre Bente

**Comment by Hypsibius** [comment]
@bubir
(Ingunn er i mellomtida godt tilbake :-) )

**Comment by danmichaelo** [comment]
Bra :) Hvis jeg forsto Bente riktig så landet hun etter litt frem og tilbake (særlig for folkemedisin) på at det i og for seg var like greit å slå sammen alle fire i et samlingsperspektiv, selv om det teknisk sett er forskjellige begreper.

**Comment by danmichaelo** [comment]
Og så var det `Naturmedisin` da (#912).. 

Tenkte først at Naturmedisin kanskje var avgrenset til medisinplanter og sånt, men ser at vi har et eget emneord for det: Medisinplanter BF Medisinske planter BF Legeurter BF Urtemedisin BF Legeplanter

**Comment by Hypsibius** [comment]
Og endå har vi denne: https://github.com/realfagstermer/realfagstermer/issues/743

Held på det eg sa der: Naturmedisin brukast om alternativ medisin, ikkje om medisinar vi finn i naturen som vi kan dokumentere fungerer.
